### DISCUSSION TITLE
When attempting to deploy a sensor, or restart Zeek on a sensor, they error out with the same issue:

### DISCUSSION THREAD

**Discussion by EchoGangster** [Security-Onion-Solutions/securityonion#4397] [Unsupported Versions]
Question to Security Onion: Do we need to decouple MISP from Zeek to restart it successfully, or will this require a rebuild of the sensors? If we do rebuild the sensors, we likely still need to take MISP off the stack. 

![Zeek_MISP Crash](https://user-images.githubusercontent.com/34453426/120666768-e7a21200-c49d-11eb-83a8-abb7a668d46e.PNG)

**Comment by weslambert** [comment]
When you say `decouple MISP from Zeek`, what kind of configuration do you currently have in place?

**Reply by EchoGangster** [reply]
Thank you for the response. We have MISP incorporated into our suite and when we had a sensor failover it would fail to restart because of intel MISP .dat file causing an error. One of our guys found a solution and had reached out to the MISP folks for permanent fix action.
